instrumentation/spring/spring-webflux-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/server/RouteOnSuccessOrError.java

```diff
@@ -41,7 +41,8 @@ public void accept(HandlerFunction<?> handler, Throwable throwable) {
 
           Span serverSpan = context.get(BaseTracer.CONTEXT_SERVER_SPAN_KEY);
           if (serverSpan != null) {
-            serverSpan.updateName(parseRoute(predicateString));
+            serverSpan.updateName(
+                BaseTracer.getApplicationRoot(context) + parseRoute(predicateString));
```

Comment:
I didn't see how to get context path directly here, but will look at it closer tomorrow

Reply:
It looks like this will work on all webflux, as suspicious as it may appear

`((RequestPath) serverRequest.pathContainer()).contextPath()`

Reply:
updated